apps/src/templates/progress/DetailProgressTable.jsx

```diff
@@ -6,13 +6,13 @@ import { levelType, lessonType } from './progressTypes';
  * A component that shows progress in a course with more detail than the summary
  * view
  */
-const DetailProgressTable = React.createClass({
-  propTypes: {
+export default class DetailProgressTable extends React.Component {
```

Comment:
I've seen this way and `extends Component` with an import of Component similar to the import of PropTypes. Do we prefer one way over the other?

Reply:
I think we're currently good with either approach - unlike `PropTypes`, `Component` is staying in the `React` package for the foreseeable future, so I don't have strong feelings about importing it separately.

For what it's worth, I've been sticking with `React.Component` just because it creates an obvious use of `React`.  It's required to have `React` in scope when writing JSX, but it's not unusual for there to be no obvious use of the import, which can be confusing to people at first.